### ISSUE TITLE
Zine physical distribution plan

### ISSUE THREAD

**Issue by wayspurrchen** [RobotArmsClub/RobotArmsClub#10]
@flipactual is interested in picking up physical distribution of the zine. Some things to figure out at a glance (please add more as you think of them):
- Is this just an at-events distribution plan?
- If we want to accept addresses and mail the zine, how do we keep track of user data?
- How do we charge users?
- How much should we charge users / how much would it cost to print?

There are probably services out there that can help us with this.

**Comment by tek-witch** [comment]
http://www.magcloud.com/

**Comment by flipactual** [comment]
I'd rather do it zine style (xerox) unless folks are opposed.

**Comment by tek-witch** [comment]
No reason not to do both.  If you want to do xerox, print the PDF out and run it down to Kinkos or whatever the modern equivalent is.

The idea behind having all the code and infrastructure behind the project is that it's available in multiple formats, right?  So if it's available as POD to people who want it, it doesn't detract from the enjoyment you'll get out of doing it manually and handing it to people.

**Comment by flipactual** [comment]
Yeah, I just don't want to be responsible for any part of the POD chain.

I am planning to use [blurb](http://www.blurb.com) for another project tho so there's a link!

**Comment by tek-witch** [comment]
I can dig it.  I don't personally want anything to do with paper cuts and staples and handing out objects to randos.  

All we do is upload the PDF to magcloud (or equivalent), check a few boxes, and we're done.  If we were _only_ going to do this xerox zine-style then this whole github trip would be pointless, you could do it all with a typewriter, scissors, and a sharpie.

**Comment by flipactual** [comment]
@exprat I'm not sure I understand. This issue is for tracking a potential physical distro. Other distribution methods are tracked in other issues.

**Comment by tek-witch** [comment]
POD is a physical distribution method.  It puts real objects in people hands, and covers all the questions in the OP, aside from events.

If this is solely an issue related to some other specific physical distribution thing and I misunderstood the goal, my bad, I'll leave you to it do it in whatever way seems most effective.

**Comment by flipactual** [comment]
Yeah, I was addressing this

> If we were only going to do this xerox zine-style then this whole github trip would be pointless, you could do it all with a typewriter, scissors, and a sharpie.

The GitHub repositories are for tracking our work/what we'll be publishing regardless of how it's published. Doing any form of physical publication would be in addition to doing the digital releases, not in place of. Physical release only was never on the table.

**Comment by tek-witch** [comment]
Gotcha.  This was where you lost me:

> I'd rather do it zine style (xerox) unless folks are opposed.

What you meant and what I thought you meant were different. Production of a xerox zine doesn't require any of this infrastructure at all, so I was confused by your response.

Anyway, rock on.

**Comment by sonyamann** [comment]
In the past I made a couple of semi-speculative budgets for printing physical zines, which may or may not be helpful.

1) Spreadsheet: https://docs.google.com/spreadsheets/d/1I0Tk4_i0urgZqZvO7FKOLgaozSr_TDKxL0AYAPrR13Q/edit?usp=sharing

2) Summary: https://docs.google.com/document/d/1ka-N3cK8KxhblAttFVXP3Vq2jnz-glhEx9P-wjFEiKk/edit?usp=sharing
